config/routes.rb

```diff
@@ -1,4 +1,16 @@
 Rails.application.routes.draw do
+  get 'students/index'
```

Comment:
Don't we want root at the top?   And since we are using all the actions for students, can we just do this?:

`resources: students`